site/content/docs/manifest/scheduled-job.md

```diff
@@ -0,0 +1,129 @@
+
+<a id="on" href="#on" class="field">`on`</a> <span class="type">Map</span>  
+The configuration for the event that triggers your job.
+
+<span class="parent-field">on.</span><a id="on-schedule" href="#on-schedule" class="field">`schedule`</a> <span class="type">String</span>  
+You can specify a rate to periodically trigger your job, supported entries:
+
+* @yearly
+* @monthly
+* @weekly
+* @daily
+* @hourly
+* @every {duration} (For example, "1m", "5m") 
+* rate({duration}) based on CloudWatch's [rate expressions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#RateExpressions) 
+
+Alternatively, you can specify a cron schedule if you'd like to trigger the job at a specific time:  
+
+* cron({fields}) based on CloudWatch's [cron expressions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions)
+
+<div class="separator"></div>
+
+<a id="cpu" href="#cpu" class="field">`cpu`</a> <span class="type">Integer</span>  
+Number of CPU units for the task. See the [Amazon ECS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html) for valid CPU values.
+
+<div class="separator"></div>
+
+<a id="memory" href="#memory" class="field">`memory`</a> <span class="type">Integer</span>  
+Amount of memory in MiB used by the task. See the [Amazon ECS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html) for valid memory values.
+
+<div class="separator"></div>
+
+<a id="retries" href="#retries" class="field">`retries`</a> <span class="type">Integer</span>  
+The number of times to retry the job before failing.
+
+<div class="separator"></div>
+
+<a id="timeout" href="#timeout" class="field">`timeout`</a> <span class="type">Duration</span>  
+How long the job should run before it aborts and fails. You can use the units: `h`, `m`, or `s`.
+
+<div class="separator"></div>
+
+<a id="variables" href="#variables" class="field">`variables`</a> <span class="type">Map</span>   
+Key-value pairs that represents environment variables that will be passed to your service. Copilot will include a number of environment variables by default for you.
+
+<div class="separator"></div>
+
+<a id="secrets" href="#secrets" class="field">`secrets`</a> <span class="type">Map</span>   
+Key-value pairs that represents secret values from [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) that will passed to your service as environment variables securely. 
+
+<div class="separator"></div>
+
+<a id="environments" href="#environments" class="field">`environments`</a> <span class="type">Map</span>  
+The environment section lets you overwrite any value in your manifest based on the environment you're in. In the example manifest above, we're overriding the count parameter so that we can run 2 copies of our service in our prod environment.
```

Comment:
```suggestion
The environment section lets you overwrite any value in your manifest based on the environment you're in. In the example manifest above, we're overriding the count parameter so that we can run two copies of our job in our prod environment.
```

Reply:
oh derp the `count` field doesn't even exist for scheduled jobs :D I updated it to override `cpu` instead